### PR TITLE
[onert-micro] Reduce Relu code duplication

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr reduces code duplication for Relu.

for #13272

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/onert-micro/src/execute/kernels/ReluCommon.cpp [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      const int flat_size = input_shape.flatSize();
+
+      status = pal::ReLUCommon(flat_size, input_data_float, output_data_float, 0.0f, is_relu_6);
+    }
+    break;
+#endif // DIS_FLOAT
+    default:
+    {
+      status = UnsupportedType;
+      assert(false && "Unsupported type.");
```
Comment (chunseoklee) [inline onert-micro/onert-micro/src/execute/kernels/ReluCommon.cpp:87]:
```suggestion
      assert(false && "Unsupported type.");
      break;
```

Comment (chunseoklee) [reply onert-micro/onert-micro/src/execute/kernels/ReluCommon.cpp:87]: static analyzer complains about missing break.

**Review by chunseoklee — DISMISSED** [review]

File: onert-micro/onert-micro/src/execute/kernels/ReluCommon.cpp (hunk 1 above)
Comment (BalyshevArtem) [reply onert-micro/onert-micro/src/execute/kernels/ReluCommon.cpp:92]: Done

**Comment by BalyshevArtem** [comment]
added LeakyRelu

**Review by Torrero — APPROVED** [review]
LGTM

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**